polling/periodic_sequence/child_workflow.go

```diff
@@ -0,0 +1,39 @@
+package periodic_sequence
+
+import (
+	"time"
+
+	"go.temporal.io/sdk/workflow"
+)
+
+type ChildWorkflowParams struct {
+	SingleWorkflowAttempts int
+	PollingInterval        time.Duration
+}
+
+func ChildWorkflow(ctx workflow.Context, params ChildWorkflowParams) (string, error) {
```

Comment:
I would rename it something like "PollingChildWorkflow". Just Child doesn't convey its purpose.

Reply:
👍